client/src/components/Chat/Message.js

```diff
@@ -0,0 +1,101 @@
+import React from 'react';
+import {
+    Avatar,
+    Box,
+    Container,
+    Typography,
+} from '@material-ui/core';
+import { makeStyles } from '@material-ui/styles';
+import { msToTimeAgo } from '../../utils/convertTimeStamps';
+
+const useStyles = makeStyles(theme => ({
+    container: {
+        padding: theme.spacing(1)
+    },
+    received_container: {
+        background: theme.palette.primary.gradient,
+        borderTopRightRadius: 20,
+        borderTopLeftRadius: 20,
+        borderBottomRightRadius: 20,
+        display: 'flex',
+        justifyContent: 'center',
+        alignItems: 'center',
+        width: 'auto',
+        maxWidth: 500,
+    },
+    sent_container: {
+        background: '#e6edff',
+        borderTopRightRadius: 20,
+        borderTopLeftRadius: 20,
+        borderBottomLeftRadius: 20,
+        display: 'flex',
+        justifyContent: 'center',
+        alignItems: 'center',
+        width: 'auto',
+        maxWidth: 500,
+        justifySelf: 'flex-end'
+    },
```

Comment:
Just curious as to why it's 2 containers. I was thinking about the logic here and you have them running through "isReceived" check. Shouldn't it just check to see if it is the current user and then position it accordingly? If the two containers is easier to accomplish this then it's understandbale. I was just thinking about the logic being put into one container.